runtime/src/blockhash_queue.rs

```diff
@@ -130,7 +130,7 @@ impl BlockhashQueue {
     )]
     #[allow(deprecated)]
     pub fn get_recent_blockhashes(&self) -> impl Iterator<Item = recent_blockhashes::IterItem> {
-        (&self.ages).iter().map(|(k, v)| {
+        (self.ages).iter().map(|(k, v)| {
```

Comment:
nit: This change doesn't seem related to this PR. Is this intentional?

Reply:
This error shows up in my local clippy run. But it looks like on CI, clippy is happy with that. 
```
> cargo clippy -Zunstable-options -p solana-runtime -- --deny=warnings --deny=clippy::integer_arithmetic --no-deps
    Blocking waiting for file lock on build directory
   Compiling solana-runtime v1.10.1 (D:\mydoc\git\hy_solana\runtime)
error: this expression borrows a value the compiler would automatically borrow
   --> runtime\src\blockhash_queue.rs:133:9
    |
133 |         (&self.ages).iter().map(|(k, v)| {
    |         ^^^^^^^^^^^^ help: change this to: `self.ages`
    |
    = note: `-D clippy::needless-borrow` implied by `-D warnings`
    = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

error: could not compile `solana-runtime` due to previous error
```
wonder if you have any idea for this?

Reply:
I would just remove this change from the PR.

You can keep the change locally to appease clippy, or open another PR for just that change too. This way it'll keep this PR as small as possible.